docker-compose.yml

```diff
@@ -0,0 +1,15 @@
+# Set the version of docker compose to use
```

Comment:
It might make sense to launch the Next.js app in the `docker-compose.yml` as well. eg. provide a `Dockerfile` that builds and runs the Next.js app, include the Dockerfile as a service in this `docker-compose.yml`.

Reply:
I'd like to avoid that if possible because of how much memory Docker containers use. If we need to do that for GitHub actions we can though (but I don't think we'd do it here, not sure).

Reply:
Okay. I was just thinking it would be simpler to understand what's going on, although I agree that the tradeoff of using less memory while testing is the right call.